packages/zero-react/src/use-query.test.ts

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Can we not do this? You can just add it on the actual line that you want to allow any.